a.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
+      <link rel="stylesheet" href="STYLE.CSS"/>
+    <title>you tube project</title>
+</head>
+<body>
+    <header>
+    
+        <div class="logo-left">
+        <i class="material-icons">menu</i>
+        <img src="https://th.bing.com/th/id/OIP.WXN1s0dF3oBlOCFVqwSVBgHaEH?w=281&h=180&c=7&r=0&o=5&pid=1.7" width="100"/>
+      
+      </div>
+      <div class="search">
+        <form action="">
+          <input type="text" placeholder="search"/>
+          <button>
+              <i class="material-icons">search</i>
+          </button>
+      </form>
+      <i class="material-icons">mic</i>
+      </div>
+      <div class="header_icons">
+        
+        <i class="material-icons">videocam</i>
+        <i class="material-icons">apps</i>
+        <i class="material-icons">notifications</i>
+        <i class="material-icons">account_circle</i>
+    </div>
+
+    </header>
+    <main>
+     
+        <div class="content">
+          <div class="thumbnails">
+            <div class="col">
+             <div class="small-img-row">
+               <div class="small-img">
+                 <img src="yellow ladybird.jpg">
+               </div>
+               <p>yellow with black dottes<br>can also fly</p>
+             </div>
+           </div>
+           <div class="thumbnails">
+             <div class="col">
+              <div class="small-img-row">
+                <div class="small-img">
+                  <img src="red ladybird.jpeg">
+                </div>
+                <p>yellow with black dottes<br>can also fly</p>
+              </div>
+            </div>
+            <div class="thumbnails">
+             <div class="col">
+              <div class="small-img-row">
+                <div class="small-img">
+                  <img src="whitedotted.jpeng.jpg">
+                </div>
+                <p>yellow with black dottes<br>can also fly</p>
+              </div>
+            </div>
```

Comment:
kindly wrapp all this section with aside tag for better styling on css

Reply:
- [ ] Hey @jemutakiplagat, Great one here also with this review, however, you've pointed out what you want the code author to do but you've not pointed out why they should do what you're proposing. I think it will make your review process more understandable if you add the reason behind your proposed change as this is one key ingredient in code review to bring clarity to the code author. Kindly explain why they should do what you've asked them to do, you can also add links to external documentation to help the code author understand better.

Reply:
- [ ] Kindly make sure to implement the changes asked by the previous reviewer by giving your review again with keeping the points in mind from [here](https://github.com/jemutakiplagat/review-project/pull/1/files#r862679113).